openedx/adg/lms/webinars/models.py

```diff
@@ -331,13 +365,41 @@ def get_webinar_update_recipients_emails(self):
             )
         )
 
+    @classmethod
+    def clone(cls, webinar_to_clone):
+        """
+        Clone the given webinar as draft
+
+        Arguments:
+            webinar_to_clone (Webinar): Webinar that we want to clone
+        """
+        old_co_hosts = webinar_to_clone.co_hosts.all()
+        old_panelists = webinar_to_clone.panelists.all()
+
+        webinar_to_clone.pk = None
+        webinar_to_clone.id = None
```

Comment:
`webinar_to_clone .pk = None` would do it. No need to make id=None

Reply:
We do need to make id none as we are inheriting from TimeStampedModel here. Take a look here: https://docs.djangoproject.com/en/dev/topics/db/queries/#copying-model-instances